### ISSUE TITLE
Change --since to ignore whitespace differences

### ISSUE THREAD

**Issue by dkubb** [mbj/mutant#441]
Currently if you change the whitespace in a code, like fixing indentation, mutant with `--since` will consider that code modified and force you to mutation test it. I think this is outside of the spirit of the original intentions for the flag, and whitespace differences should be ignored.

The `git log` command supports the `--ignore-all-space` flag, which when added to `Mutant::Repository` methods should give the desired behaviour.

**Comment by mbj** [comment]
> I think this is outside of the spirit of the original intentions for the flag, and whitespace differences should be ignored.

Yes. Agreed. The proposed `--ignore-all-space` flag is the most "low tech" solution to the problem.

an alternative is to actually make sure the AST of the subject was changed.
